docs/src/man/getting_started.md

```diff
@@ -111,61 +123,202 @@ julia> nulls(Int, 1, 3)
 
 The `DataFrame` type can be used to represent data tables, each column of which is a vector. You can specify the columns using keyword arguments or pairs:
 
-```julia
-df = DataFrame(A = 1:4, B = ["M", "F", "F", "M"])
-df = DataFrame(:A => 1:4, :B => ["M", "F", "F", "M"])
+```jldocttest
+julia> using DataFrames
+
+julia> df = DataFrame(A = 1:4, B = ["M", "F", "F", "M"])
+4×2 DataFrames.DataFrame
+│ Row │ A │ B │
+├─────┼───┼───┤
+│ 1   │ 1 │ M │
+│ 2   │ 2 │ F │
+│ 3   │ 3 │ F │
+│ 4   │ 4 │ M │
+
+julia> df == DataFrame(:A => 1:4, :B => ["M", "F", "F", "M"])
```

Comment:
Better just create the data frame and let the user see it gives the same result (no need to define `df` in either of these lines).